Core/Currency.cs

```diff
@@ -1,7 +1,7 @@
 ﻿namespace Core
 {
-    public enum Currency
-    {
-        UAH
-    }
+	public enum Currency
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Replace all tab characters in this file by sequences of white-spaces.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=4931949)